content/broker/broker-transition/getting-started/_index.en.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Getting started with Altinn Broker Transition Solution
+linktitle: Getting Started
+description: How to get started using the Broker Transition solution from Altinn 2 to Altinn 3
+tags: [solution, broker, transition, guide]
+toc: false
+weight: 1
+---
+
+## ServiceOwners
+In order to use the Broker Transition solution in Altinn 2 to create, upload and retrieve file metadata, a service owner must complete the following steps.
+1. Have an existing Altinn 2 Broker Service.
+2. Have or create a corresponding Altinn 3 Broker Resource. 
+See how to get started with Altinn Broker [here](../../getting-started/).
+1. <!-- TODO: Set up of rights for the Altinn 3 Resource>
+<!-- TODO: Should we have links here to the different services used by service owners to create the different services? -->
+1. Request a transition setup from the Altinn 2 Service to the Altinn 3 Resource. Determine the date for when this should go live. (To minimize the risk of orphaned data in Altinn 2 storage)
```

Comment:
Is this the start of a new list, or is it a continuation of the previous? a lack of a header leads this to be weirdly formatted.
Either change numbering og add some text og header between.

Reply:
Changed the list, took away todo